src/iop/colorzones.c

```diff
@@ -2136,8 +2149,7 @@ static void _channel_tabs_switch_callback(GtkNotebook *notebook, GtkWidget *page
 
 static void _color_picker_callback(GtkToggleButton *togglebutton, dt_iop_module_t *module)
 {
-  if(gtk_toggle_button_get_active(togglebutton))
-    dt_iop_color_picker_set_cst(module, iop_cs_LCh);
+  if(gtk_toggle_button_get_active(togglebutton)) dt_iop_color_picker_set_cst(module, iop_cs_LCh);
```

Comment:
Ooops. Some automatic code re-formating by clang-format slipped in. This and all changes below will be reverted,